hydra-node/src/Hydra/Network/Ouroboros.hs

```diff
@@ -14,6 +14,7 @@ import Hydra.Prelude
 
 import Codec.CBOR.Term (Term)
 import qualified Codec.CBOR.Term as CBOR
+import Control.Concurrent (newMVar, putMVar, readMVar, takeMVar)
```

Comment:
Favor [`io-sim`'s classes](https://github.com/input-output-hk/io-sim) over `Control.Concurrent` here, they work in `STM` which is much better than IO.